tests/bot/exts/info/test_information.py

```diff
@@ -409,7 +411,7 @@ async def test_create_user_embed_uses_top_role_colour_when_user_has_roles(self):
         moderators_role = helpers.MockRole(name='Moderators')
         moderators_role.colour = 100
 
-        user = helpers.MockMember(id=314, roles=[moderators_role], top_role=moderators_role)
+        user = helpers.MockMember(id=314, roles=[moderators_role], top_role=moderators_role, colour=100)
```

Comment:
`top_role` isn't used anymore, apply this change at the other places too, not commenting everywhere.
```suggestion
        user = helpers.MockMember(id=314, roles=[moderators_role], colour=100)
```